examples/docs_beta_snippets/docs_beta_snippets/guides/components/deployments/7-component-type-list.txt

```diff
@@ -0,0 +1,9 @@
+$ cd code_locations/code-location-1 && dg component-type list
+
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ Component Type                                        ┃ Summary                                                      ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│ dagster_components.definitions                        │ Wraps an arbitrary set of Dagster definitions.               │
+│ dagster_components.pipes_subprocess_script_collection │ Assets that wrap Python scripts executed with Dagster's      │
+│                                                       │ PipesSubprocessClient.                                       │
+└───────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────────┘
```

Comment:
this is a bit minor, but in the [preview](https://dagster-docs-f7sq4v73j-elementl.vercel.app/guides/build/components#project-setup) these are kinda wide for the box that they live in, meaning we have a scroll bar

if there's some easy way to tell the command line that the window size is like 100 characters or something so that our cli can squish the text a bit more then that'd be great, but definitely not rabbit-hole worthy at the moment

Reply:
Yeah, we actually have a 120 char limit already through the `COLUMNS` env var - I'll bump it down to 96